source/opt/dead_branch_elim_pass.cpp

```diff
@@ -0,0 +1,262 @@
+// Copyright (c) 2017 The Khronos Group Inc.
+// Copyright (c) 2017 Valve Corporation
+// Copyright (c) 2017 LunarG Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "dead_branch_elim_pass.h"
+
+#include "cfa.h"
+#include "iterator.h"
+
+static const int kSpvEntryPointFunctionId = 1;
+static const int kSpvBranchCondConditionalId = 0;
+static const int kSpvBranchCondTrueLabId = 1;
+static const int kSpvBranchCondFalseLabId = 2;
+static const int kSpvSelectionMergeMergeBlockId = 0;
+static const int kSpvPhiVal0Id = 0;
```

Comment:
The structure of these names is:

kSpv<Name of Operation><NameOfOperand>

My goal was to avoid just have having magic numbers all over the code, and if at some future point an instruction changed we might have some hope of tracking its presence down.

Can you suggest a better structure? I am trying to keep their length somewhat manageable.